microlith/loki/alerting/couchbase/couchbase-rules.yaml

```diff
@@ -273,3 +273,19 @@ groups:
           title: "Large Number of Dropped Ticks Detected (node: {{ $labels.hostname }})"
           description: Couchbase Server nodes regularly send heartbeat ticks to each other. Dropped ticks are usually a sign of resource contention.
           remediation: Increase number of CPUs available to Couchbase Server or, if you are running a virtualised environment, check for VM overcommitment.
+
+      - alert: CB90067-serviceRollback
+        expr: |
+          count_over_time( {file="memcached.log"} |= "requires rollback to seqno:0" != "eq_dcpq:replication" [24h]) > 0
+        labels:
+          job: couchbase_fluent_bit
+          kind: node
+          severity: alert
```

Comment:
Does someone need to be woken up at 1am if this happens? If not, the severity should be `warning` at most.

Reply:
Will fix this